test/Microsoft.Web.LibraryManager.Vsix.Test/UI/Converters/CheckBoxAutomationNameConverterTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Web.LibraryManager.Vsix.Resources;
+using Microsoft.Web.LibraryManager.Vsix.UI.Converters;
+
+namespace Microsoft.Web.LibraryManager.Vsix.Test.UI.Converters
+{
+    [TestClass]
+    public class CheckBoxAutomationNameConverterTests
+    {
```

Comment:
All the happy paths are tested here, but this could use some more tests of unexpected inputs.